services/tinlake/actions.ts

```diff
@@ -438,13 +456,13 @@ export async function redeem(
     return loggedError(e, `Could not redeem ${trancheType}.`, '')
   }
 
-  return { status: 0 }
+  return { status: 0 } as any
```

Comment:
😢

Reply:
Fixed now by commenting `receipt` in tinlake.js :)